decorator.rb

```diff
@@ -0,0 +1,9 @@
+class Decorator < Nameable
+  def initialize()
```

Comment:
- [ ] It would be better if we assign a nameable object from params to an instance variable. Here is how we can fix it
```

def initialize(nameable)
  @nameable = nameable
end

```